os/kernel/binary_manager/binary_manager.h

```diff
@@ -35,13 +39,22 @@
  ****************************************************************************/
 #define BINARY_MANAGER_NAME        "binary_manager"           /* Binary manager thread name */
 #define BINARY_MANAGER_STACKSIZE   2048                       /* Binary manager thread stack size */
-#define BINARY_MANAGER_PRIORITY    250                        /* Binary manager thread priority */
+#define BINARY_MANAGER_PRIORITY    201                        /* Binary manager thread priority */
+
+/* Fault Message Sender Thread information */
+#define FAULTMSGSENDER_NAME           "bm_faultmsgsender"     /* Fault Message Sender thread name */
+#define FAULTMSGSENDER_STACKSIZE       1024                   /* Fault Message Sender thread stack size */
+#define FAULTMSGSENDER_PRIORITY        202                    /* Fault Message Sender thread priority */
```

Comment:
I think faultmsgsender should have highest priority in bm priority range. So it should be `BM_PRIORITY_MAX`.